src/platform/nrfconnect/wifi/WiFiManager.h

```diff
@@ -198,14 +198,18 @@ class WiFiManager
     constexpr static uint32_t kWifiManagementEvents = NET_EVENT_WIFI_SCAN_RESULT | NET_EVENT_WIFI_SCAN_DONE |
         NET_EVENT_WIFI_CONNECT_RESULT | NET_EVENT_WIFI_DISCONNECT_RESULT | NET_EVENT_WIFI_IFACE_STATUS;
 
+    constexpr static uint32_t kIPv6ManagementEvents = NET_EVENT_IPV6_PREFIX_ADD;
```

Comment:
Wouldn't it be more straightforward to react on `NET_EVENT_IPV6_ADDR_ADD` and `NET_EVENT_IPV6_ADDR_DEL`? That way, this would be correctly handled even if the address be added via other means. Then you could filter out link-local addresses in the event handler.

Reply:
Makes sense to me. Assuming that for whatever reason these are not excluding addresses assigned by SLAAC (I believe this eventing should be "source" agnostic but still).

Reply:
> Wouldn't it be more straightforward to react on NET_EVENT_IPV6_ADDR_ADD and NET_EVENT_IPV6_ADDR_DEL?
Ok, actually that was my first idea. Then I decided to use `NET_EVENT_IPV6_PREFIX_ADD` to skip the link-local address filtering part, but I get your point.